R/utils-time.R

```diff
@@ -145,3 +145,20 @@ boundDate <- function(date, dateRange, padDays=1){
   return(date)
 }
 
+#' Format Time as a label with the UTC Offset
+#'
+#' @description Formats a time datum as a text
+#' label including the UTC offset at the end.
+#' @param time The time datum to format as a text label
+formatUTCTimeLabel <- function(time){
+  formatted_time <- ""
+  
+  if(!isEmptyOrBlank(time)){
+    tzf <- format(as.POSIXct(time), "%z")
```

Comment:
did you try this for 9999 or does it never encounter that because it's just the start date?